source/.byebug_history

```diff
@@ -0,0 +1,11 @@
+continue
+next
+@url
+next
+@url
+continue
+@url.errors.first.essage
+@url.errors.full_messages
+@url.errors.each { |error| puts error.full_messages }
+@url.errors.each do |error|
+@url.errors
```

Comment:
This file should probably go in .gitignore - no reason for it to be part of the history of the repo.